src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1153,6 +1163,35 @@ public OpenLoadFlowParameters setVoltageTargetPriorities(List<String> voltageTar
         return this;
     }
 
+    public boolean isTransformerVoltageControlStable() {
+        return transformerVoltageControlStable;
+    }
+
+    public OpenLoadFlowParameters setTransformerVoltageControlStable(boolean transformerVoltageControlStable) {
+        this.transformerVoltageControlStable = transformerVoltageControlStable;
+        return this;
+    }
+
+    /**
+     * Returns in KV the tension below wich groups controlling tension are forced to PQ
+     * during tap computation.
+     * @return The HT limit or -1 if the limit is determined automatically
+     */
+    public int getTransformerVoltageControlThtLimit() {
+        return transformerVoltageControlThtLimit;
+    }
+
+    /**
+     * Sets the tension below wich groups controlling tension are forced to PQ
+     * during tap computation.
+     * @param transformerVoltageControlThtLimit the limit in KV or -1 the limit
+     *                                         shoud be determined automatically.
+     */
```

Comment:
same / french

Reply:
should have disappeared with the merge and replaced by the new parameter name.